packages/@o3r/eslint-config-otter/package.json

```diff
@@ -10,7 +10,7 @@
     "linter"
   ],
   "scripts": {
-    "build": "yarn cpy '{index,template,fast}.js' package.json dist && yarn postbuild",
+    "build": "yarn cpy '{index,base,template,fast,jasmine,jasmine-fast}.js' package.json dist && yarn postbuild",
```

Comment:
Should we take the opportunity of this PR to add `exports` for the non-default js files?